debian/copyright

```diff
@@ -27,11 +89,16 @@ Copyright: 2016-2017 Bowtie AB
            2018-2020 Jack Marsh
 License: MIT
 
-Files: pynicotine/metadata/tinytag.py
-Copyright: 2014-2022 Tom Wallroth
-           2020-2022 Nicotine+ Team
+Files: pynicotine/metadata/*
+Copyright: 2014-2021 Tom Wallroth
+           2020-2021 Mat (mathiascode) <mail@mathias.is>
 License: MIT
 
+Files: debian/*
+Copyright: 2022 Francois Mazen <francois@mzf.fr>
+                Bastian Germann <bage@debian.org>
```

Comment:
This is only true for the official Debian package, and need to be changed for the package that is provided by nicotine team.

Reply:
I disagree, because both of you have the option of adding yourself or being added by us into AUTHORS.md and about.py at any point should you wish to do so, and such a request is very likely to be approved. Therefore, it is easier just to leave the entries in.

Reply:
@slook Downstream packaging is different from ours, and mine and Kip's names are covered by the general file copyright already.